opm/simulators/wells/StandardWellAssemble.cpp

```diff
@@ -264,48 +256,6 @@ assembleZFracEq(const EvalWell& cq_s_zfrac_effective,
 
 #define INSTANCE(Dim,...) \
 template class StandardWellAssemble<BlackOilFluidSystem<double,BlackOilDefaultIndexTraits>,__VA_ARGS__,double>; \
-template void \
-StandardWellAssemble<BlackOilFluidSystem<double,BlackOilDefaultIndexTraits>,__VA_ARGS__,double>:: \
```

Comment:
Hm, removing this leaves the previous line (`template class StandardWellAssemble<...>`) ending in a backslash character (`\`).  It is benign in this case since the following line is blank, but I think it might be preferable to remove the trailing backslash here just to be on the safe side.

Reply:
indeed.